### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by keradus**

### CODE REVIEW

File: Symfony/CS/Fixer/Symfony/OperatorsSpacesFixer.php [hunk 1]
```diff
@@ -76,6 +76,6 @@ public function fix(\SplFileInfo $file, $content)
      */
     public function getDescription()
     {
-        return 'Binary operators should be surrounded by at least one space.';
```
Comment (GrahamCampbell) [inline Symfony/CS/Fixer/Symfony/OperatorsSpacesFixer.php]: Sad that this changed. Was a massive breaker.

Comment (keradus) [inline Symfony/CS/Fixer/Symfony/OperatorsSpacesFixer.php]: cc @SpacePossum

Comment (GrahamCampbell) [inline Symfony/CS/Fixer/Symfony/OperatorsSpacesFixer.php]: I'd suggest we make that part configurable. Should be pretty easy.

Comment (GrahamCampbell) [inline Symfony/CS/Fixer/Symfony/OperatorsSpacesFixer.php]: Also, this fixer technically conflicts with the array alignment fixers now.

Comment (GrahamCampbell) [inline Symfony/CS/Fixer/Symfony/OperatorsSpacesFixer.php]: Really bad to have one fixer changing stuff, then another one reverting the changes.

Comment (SpacePossum) [inline Symfony/CS/Fixer/Symfony/OperatorsSpacesFixer.php]: duplicate report https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/2151#issuecomment-241357875

**Pull request closed without merging**